docs/firebase.md

```diff
@@ -308,3 +308,23 @@ app:
 Open the web app by visiting the URL printed by the `deploy` command. The app
 requires you to sign in with a Google account, after which you can initiate
 endpoint requests.
+
+## Developing using Firebase Local Emulator Suite
+
+Firebase offers a suite of emulators for local development and you can use Genkit with [Firebase Local Emulator Suite](https://firebase.google.com/docs/emulator-suite).
+
+To use Genkit with the Firebase Emulator Suite, start the the Firebase emulators like this:
+
+```bash
+GENKIT_ENV=dev firebase emulators:start --inspect-functions
+```
+
+this will run your code in the emulator and run genkit framework in dev mode which launches and exposes the Genkit reflection API (but not the Dev UI).
```

Comment:
```suggestion
This will run your code in the emulator and run the Genkit framework in development mode, which launches and exposes the Genkit reflection API (but not the Dev UI).
```